src/test/java/com/maddisportfolio/user/UserServiceTest.java

```diff
@@ -366,6 +366,40 @@ public void testSearchUsersIsFriendsByRecipient(){
         Assert.assertTrue(kupo.isFriendsWithLoggedInUser());
     }
 
+    @Test
+    public void testDeclineRequestToLoggedInUser(){
```

Comment:
This method's named "testDeclineRequest..." but the method you're testing is searchUsers. I'd rename the method to describe the method & scenario you're testing